app/installer.sh

```diff
@@ -23,10 +23,13 @@ DIR="$( cd "$( dirname "$0" )" >/dev/null 2>&1 && pwd )"
 : "${VERSION:=}"
 : "${ARCH:=}"
 : "${PRODUCT_NAME:=Kuma}"
+: "${REPO:=kumahq/kuma}"
+: "${BRANCH:=master}"
 : "${LATEST_VERSION:=https://kuma.io/latest_version}"
-: "${REPO_PREFIX:=kuma}"
 : "${CTL_NAME:=kumactl}"
 
+REPO_PREFIX=$(echo $REPO | sed 's/.*\///')
```

Comment:
```suggestion
REPO_PREFIX=$(echo $REPO | sed 's|.*/||')
```

I find this a bit more readable but up to you to ignore this

Reply:
I like this better as well, changing it.